pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -113,3 +113,12 @@ func (c *CephObjectStore) GetStatusConditions() *[]Condition {
 func (z *CephObjectZone) GetStatusConditions() *[]Condition {
 	return &z.Status.Conditions
 }
+
+// String returns an addressable string representation of the EndpointAddress.
+func (e *EndpointAddress) String() string {
+	// hostname is easier to read, and it is probably less likely to change, so prefer it over IP
+	if e.Hostname != "" {
+		return e.Hostname
+	}
+	return e.IP
+}
```

Comment:
Using a pared down version of the `EndpointAddress` allows us to more easily stringify the endpoint. It's either the specified hostname or IP. Nothing fancy.